examples/kotlin-diktat/sarif-actual/save-warnings-actual.sarif

```diff
@@ -0,0 +1,155 @@
+{
```

Comment:
I added this config for consistency: by default warn plugin in sarif mode will look into stdout of tool for `.sarif` report
But, it also possible to provide such report via file, like this


In aim to avoid misunderstandings, I add this config, however it's not required as been said